### DISCUSSION TITLE
how to change the Layout of ArgsTable for `<Controls/>`？

### DISCUSSION THREAD

**Discussion by Flcwl** [storybookjs/storybook#23188] [Help]
I want to change the tableHeader title in ArgsTable?
![image](https://github.com/storybookjs/storybook/assets/26050644/a8059626-0dc0-440a-983b-24526280d8bc)

My expected result like this:
![image](https://github.com/storybookjs/storybook/assets/26050644/97a82847-6b7c-4954-91cb-3d3cffe696e3)

**Comment by Flcwl** [comment]
I just want update title to support i18n.

**Comment by vvs09** [comment]
This should help https://storybook.js.org/docs/angular/api/doc-block-argtypes

**Reply by Flcwl** [reply]
I think the `argtypes` can't do it. I want to do two things.
1. Replace form english into chinese or others for table header title of docs. 
2. insert docs table new column as `type`.
